code/WorkInProgress/Ported/policetape.dm

```diff
@@ -144,7 +144,7 @@
 		for(var/atom/A in T) //Check to see if there's anything solid on the tape's turf (it's possible to build on it)
 			if(A.density)
 				return
-		M:loc = T
+		M:forceMove(T)
```

Comment:
would it not be better to typecast M as var/mob/M instead of "M as mob" and replace the ":" in this?